instrumentation/apache-elasticjob-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apacheelasticjob/v3_0/ElasticJobExperimentalAttributeExtractor.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apacheelasticjob.v3_0;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
+import javax.annotation.Nullable;
+
+class ElasticJobExperimentalAttributeExtractor
+    implements AttributesExtractor<ElasticJobProcessRequest, Void> {
+
+  private static final AttributeKey<String> ELASTICJOB_JOB_NAME =
+      AttributeKey.stringKey("scheduling.apache-elasticjob.job.name");
+  private static final AttributeKey<String> ELASTICJOB_TASK_ID =
+      AttributeKey.stringKey("scheduling.apache-elasticjob.task.id");
+  private static final AttributeKey<Long> ELASTICJOB_ITEM =
+      AttributeKey.longKey("scheduling.apache-elasticjob.item");
```

Comment:
the meaning behind this `scheduling.apache-elasticjob.item` attribute was not obvious to me until I looked at how it's used as a sorting key in the tests. In my mind, having `.index` or `.position` or `.order` or something would be more intuitive to indicate what it means, but I'm not familiar with this domain to know if those would make sense.